src/lib/validators.ts

```diff
@@ -99,13 +122,22 @@ const validateFieldConfig = (data: unknown): Result => {
 		}
 	}
 
+	if (data.presentation !== undefined && !isObject(data.options)) {
+		return error(lang(EN_PRESET_IS_NOT_OBJECT));
```

Comment:
The argument to `isObject` here should surely be `data.presentation`? The lang constant `EN_PRESET_IS_NOT_OBJECT` should probably also be updated from `PRESET` to `PRESENTATION`.

Reply:
@tristan-go Fixed bellow!

Reply:
Awesome, that's fixed it for me! I'll do one last look over, but I think it's ready to merge.